docs/logicapps-ai-course/02_build_conversational_agents/09-connect-agents-a2a-protocol.md

```diff
@@ -0,0 +1,492 @@
+
+### Key features
+- **Interoperability**: Aligns with A2A protocol for compatibility with other platforms.
+- **Multi-turn conversations**: Supports back-and-forth interactions.
+- **Real-time streaming**: Provides live status updates.
+- **Conversational memory**: Maintains context across sessions.
+- **Authentication options**: API key and OAuth.
+
+### Goals
+- Easily incorporate Azure Logic Apps agents into multi-agent solutions.
+- Replace code-heavy agents with minimal changes.
+
+## Trigger parameters and authentication for conversational agents
+
+### Fields in the conversational agent trigger
+
+- **Agent URL**: All communication with a Logic Apps conversational agent happens through this **Agent URL**, which maps directly to the A2A specification's serviceEndpoint. This URL serves as the entry point for all A2A-compliant API calls, such as message/send, tasks/get, and message/stream. 
+- **Agent API key**: The API key (also referred to as the developer key) must be included in the X-API-Key header for any communication with the agent.
+- **Name**: User provided name for the agent. Shows up in the `name` property of the Agent Card (explained in next section).
+- **Description**: User provided description for the agent. Shows up in the `description` property of the Agent Card (explained in next section).
+
+![Conversational Agent Trigger parameters](./media/10_Connect_your_agents_using_a2a_protocol/A2ATrigger.png)
+
+### Authentication methods
+
+Logic App conversational agents support two authentication methods:
+
+- **API key-based authentication**: This is the default method. The API key (also referred to as the developer key) must be included in the X-API-Key header for any communication with the agent.
+- **App Service authentication (EasyAuth)**: If EasyAuth is configured on the Logic App (Standard), it takes precedence over API key-based authentication. This method is typically used when OAuth flows or on-behalf-of (OBO) scenarios are required, such as interacting with connectors that require user context (explained in [Module 05 — Add user context to your tools](./05-add-user-context-to-tools.md)).
+
+## Agent discovery and agent card
+
+Each Azure Logic Apps conversational agent exposes an [**agent card**](https://a2a-protocol.org/latest/specification/#5-agent-discovery-the-agent-card) defined by the [A2A specification](https://a2a-protocol.org/latest/specification/), a JSON document that acts as a digital business card. This card is essential for discovery and initiating interaction.
```

Comment:
How does the customer use this information in the context of this tutorial?
Reshuffling the word order to put the "why care" part earlier.

```suggestion
In Azure Logic Apps, each conversational agent exposes an *agent card* that acts as a digital business card and is essential to discovering the agent and initiating interactions. The agent card is a JSON document that's defined by the [A2A specification](https://a2a-protocol.org/latest/specification/#5-agent-discovery-the-agent-card).

```